### ISSUE TITLE
Docker deployed version shown in 1.67 is 1.66

### ISSUE THREAD

**Issue by bbz404** [librenms/docker#125]
#### Steps to reproduce this issue

1. Deploy librenms install via kubernetes\docker
2. Check "about librenms" from gear menu
3. Version shows 1.66

#### Expected behaviour

> Version shown should be 1.67

#### Actual behaviour

> Version shown is 1.66

### Configuration

* Docker version (type `docker --version`) : Docker version 19.03.12, build 48a66213fe
* Docker compose version if applicable (type `docker-compose --version`) : 
* Platform (Debian 9, Ubuntu 18.04, ...) : Ubuntu 20.04
* System info (type `uname -a`) : 5.4.0-47-generic #51-Ubuntu SMP Fri Sep 4 19:50:52 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
* Include all necessary configuration files : `docker-compose.yml`, `.env`, ... 
>  I have deployed LibreNMS via k8s, relevant section from deployment config:
       containers:
        - name: librenms
          image: 'librenms/librenms:1.67''

### Docker info

```
> Client:
 Debug Mode: false

Server:
 Containers: 53
  Running: 32
  Paused: 0
  Stopped: 21
 Images: 36
 Server Version: 19.03.12
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: systemd
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc version: dc9208a3303feef5b3839f4323d9beb36df0a9dd
 init version: fec3683
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 5.4.0-47-generic
 Operating System: Ubuntu 20.04.1 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 6
 Total Memory: 7.749GiB
 Name: kube1
 ID: QAKQ:HZXN:PSKC:WO5W:26DZ:RPWJ:MNJE:SBNW:SURN:4RMQ:RVGL:5TSD
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

WARNING: No swap limit support
`
```

### Logs

```
> Container logs (set LOG_LEVEL to debug if applicable)

Output of docker image ls
REPOSITORY                                       TAG                 IMAGE ID
librenms/librenms                                1.67                bc7588459859

docker image inspect librenms/librenms:1.67
[
    {
        "Id": "sha256:bc75884598594745eb950b980e8074229bb63f64ad7c9dd60cfb2337e690d85d",
        "RepoTags": [
            "librenms/librenms:1.67"
        ],
        "RepoDigests": [
            "librenms/librenms@sha256:afc8bd574c8c00939ab2ef5b73f09176068ec2742142721a6aea2fdad6f2317f"
        ],
        "Parent": "",
        "Comment": "buildkit.dockerfile.v0",
        "Created": "2020-09-03T22:12:54.339210906Z",
        "Container": "",
        "ContainerConfig": {
            "Hostname": "",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": null,
            "Cmd": null,
            "Image": "",
            "Volumes": null,
            "WorkingDir": "",
            "Entrypoint": null,
            "OnBuild": null,
            "Labels": null
        },
        "DockerVersion": "",
        "Author": "",
        "Config": {
            "Hostname": "",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "ExposedPorts": {
                "514/tcp": {},
                "514/udp": {},
                "8000/tcp": {}
            },
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "S6_BEHAVIOUR_IF_STAGE2_FAILS=2",
                "LIBRENMS_VERSION=1.67",
                "LIBRENMS_PATH=/opt/librenms",
                "LIBRENMS_DOCKER=1",
                "TZ=UTC",
                "PUID=1000",
                "PGID=1000"
            ],
            "Cmd": null,
            "ArgsEscaped": true,
            "Image": "",
            "Volumes": {
                "/data": {}
            },
            "WorkingDir": "/opt/librenms",
            "Entrypoint": [
                "/init"
            ],
            "OnBuild": null,
            "Labels": {
                "maintainer": "CrazyMax",
                "org.opencontainers.image.created": "2020-09-03T22:09:10Z",
                "org.opencontainers.image.description": "LibreNMS",
                "org.opencontainers.image.licenses": "MIT",
                "org.opencontainers.image.revision": "f68d1b28",
                "org.opencontainers.image.source": "https://github.com/librenms/docker",
                "org.opencontainers.image.title": "LibreNMS",
                "org.opencontainers.image.url": "https://github.com/librenms/docker",
                "org.opencontainers.image.vendor": "CrazyMax",
                "org.opencontainers.image.version": "1.67"
            }
        },
        "Architecture": "amd64",
        "Os": "linux",
        "Size": 863953667,
        "VirtualSize": 863953667,
        "GraphDriver": {
            "Data": {
                "LowerDir": "/var/lib/docker/overlay2/14475b0808aeeaddb3f3a7a8c42bd728d0108d20c456ab59fa09701395764338/diff:/var/lib/docker/overlay2/bca89e13dffcc3aa3264681cfe7e740b7f3e6813dc4460bf1a4be3dc4eb80515/diff:/var/lib/docker/overlay2/3a1dc01baea6dbdc085917336612a88c30a0ed1f2f8ebc1f1d7a99e00deb383f/diff:/var/lib/docker/overlay2/8bd903941329d681506df852061fafd668aa5e45a6bacc7cac1b95e8e12fed9b/diff:/var/lib/docker/overlay2/b21e8ce82e9b055a9f5673a3ef97d009931b55ded6881eb341b3cc31538dc44b/diff:/var/lib/docker/overlay2/9af2d550c606d1b6b1ab893b27e7cab07a2c0fdedbe7ed516a8c8f214d1d9984/diff:/var/lib/docker/overlay2/a2f88d7ab5dabb9aa4c2168a6a06062eff24b8007bb815c2cfacdfe3c8a83dc5/diff:/var/lib/docker/overlay2/42ac33d9b1b8737627616cfc7b78f681e419332280b9a3d6418473298e6682db/diff:/var/lib/docker/overlay2/a068a945051825131578f212c00f9b76d9d4da2b04e0f277bd490df8b2872b5f/diff",
                "MergedDir": "/var/lib/docker/overlay2/cf27fdf632d336f57070c71897f8fc80937f8e37d2733e7e999ec370de138e02/merged",
                "UpperDir": "/var/lib/docker/overlay2/cf27fdf632d336f57070c71897f8fc80937f8e37d2733e7e999ec370de138e02/diff",
                "WorkDir": "/var/lib/docker/overlay2/cf27fdf632d336f57070c71897f8fc80937f8e37d2733e7e999ec370de138e02/work"
            },
            "Name": "overlay2"
        },
        "RootFS": {
            "Type": "layers",
            "Layers": [
                "sha256:50644c29ef5a27c9a40c393a73ece2479de78325cae7d762ef3cdc19bf42dd0a",
                "sha256:f14a0be5da2fb7cc93a1a8f1d8fbc3a120d7b0fed52a9dab87c7ede2c40fa5e0",
                "sha256:4265d9ea5ef3c25290d7755869c1e9c4bf0a68410e108fda63ed62d0f8ed7848",
                "sha256:35963347ed822dc48c0dacc1bfebaf9a8613789ff9a452d02f970f5340c1cb47",
                "sha256:4e36f6403a38e4966da6824e148fe551633c5fa1c4d0b7817b0b4832abf79f9b",
                "sha256:9e2f24e9df62577f60d05dd5ef8c5f3232a12511f8edeb920c95384ed0ce81cf",
                "sha256:60c56c81a2ac98c772cf6c1b5b0926606c797bc593fde17adee5f1ab4e00b403",
                "sha256:e2b9c20175a7c5d31f23a3512c07e6e346957bf689884f32e082aa14e8c46abb",
                "sha256:7708f2a4140fd0456e184a8d6f99eb13ef774aa8590b151dbbe31a7c183e2282",
                "sha256:c2915037f3a3ef62bfeff21be850d48fe3b3e0e0ed79709bc73cf8a2e821ea69"
            ]
        },
        "Metadata": {
            "LastTagTime": "0001-01-01T00:00:00Z"
        }
    }
]

```

**Comment by crazy-max** [comment]
@bbz404 Looks like an upstream issue. Can you confirm @murrant?

**Comment by murrant** [comment]
I cannot, the version in Version.php was 1.67 (and is now 1.68)

**Comment by crazy-max** [comment]
@murrant Looks like it's solved with 1.68

![image](https://user-images.githubusercontent.com/1951866/94742087-a680bd80-0364-11eb-9d98-56f67f6fbdac.png)
